00-open-a-shapefile.Rmd

```diff
@@ -6,15 +6,17 @@ authors: [Joseph Stachelek, Leah Wasser]
 contributors: [Megan A. Jones, Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
-packagesLibraries: [raster, rgdal]
+packagesLibraries: [rgdal, raster]
 category: 
 tags: [module-1]
 mainTag: GIS-Spatial-Data
-description: "This lesson explains the how to open and plot point, line, and polygon shapefiles in R."
+description: "This lesson explains the how to open and plot point, line and
+polygon shapefiles in `R`."
```

Comment:
hi - yes i've been working on the raster lessons. I've been rebuilding locally - have only pushed the updated MD and RMD files as i finish each file. I haven't pushed images, etc as i wanted to clean out all images on the last rebuild and push them all at once to keep the repo a bit smaller. I am half way through the last lesson however so that will happen today or tomorrow! The lessons look great! You added some great things - especially in the last lesson which i quickly realized wasn't finished!